### ISSUE TITLE
Ability to test UIActivityViewController

### ISSUE THREAD

**Issue by Nomad00** [kif-framework/KIF#635]
I've run into an issue attempting to automate the enabling of an extension (via the "More" button).

Once the share sheet has been shown, I attempt to tap the "More" button like so:

tester().tapViewWithAccessibilityLabel("More")

Which is so close to working. The problem is that there are two "More" buttons. Adding to this problem is that neither of the UIACollectionView s have accessibility labels set, so I cannot scope the search for "More". 

Perhaps I'm missing something - any thoughts would be appreciated!

![uiautomationdump](https://cloud.githubusercontent.com/assets/868020/7320227/18e41838-ea69-11e4-9929-7b4236c21496.png)

**Comment by phatmann** [comment]
I looked into this and I don't have a good solution for you. There is a lower-level KIF call  `accessibilityElementMatchingBlock:` in UIApplication-KIFAdditions. If there were a version of this method that returned all matches, then perhaps you could grab the first one or last one, but the order might be unpredictable. Are there any differences between the two buttons that you could use in the matching block to distinguish them?
